src/main/java/com/checkout/apm/sepa/MandateResponse.java

```diff
@@ -0,0 +1,50 @@
+package com.checkout.apm.sepa;
+
+import com.checkout.common.CountryCode;
+import com.checkout.common.Resource;
+import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+
+@Data
+public final class MandateResponse extends Resource {
+
+    @SerializedName("mandate_reference")
+    private String mandateReference;
+
+    @SerializedName("customer_id")
+    private String customerId;
+
+    @SerializedName("first_name")
+    private String firstName;
+
+    @SerializedName("last_name")
+    private String lastName;
+
+    @SerializedName("address_line1")
+    private String addressLine1;
```

Comment:
Only `addressLine1` is available in this response? Usually 2 addresses are returned.

Reply:
Yes, it only expects one https://docs.checkout.com/payments/payment-methods/direct-debit/sepa-direct-debit#SEPADirectDebit-CreateanewSDDpaymentsourcecreatepaymentsource